### PR TITLE
Introduction to G1ANT.Language

### PR DESCRIPTION
**Pull request by Vegebutcher**
Any corrections, comments, inputs welcome!

### CODE REVIEW

File: g1ant-language/introduction.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Introduction
+
+First thing you notice about G1ANT.Robot is that you communicate with it using scripts. These scripts are written in a special programming code called — surprise! — G1ANT.Language. You probably wonder what it is, why we chose to create it, and why we think this solution is superior to others. Here are the answers.
+
+## G1ANT.Language: What Is It?
```
Comment (WiktoriaPrusik) [inline g1ant-language/introduction.md]: is it really true that we have our own programming language that is called "G1ANT.Language"?
for me it's weird as we don't have C#.Language, Python.Language or JQuery.Language languages. 

I think that G1ANT.Robot should be a software and G1ANT a programming language.

we're going to have a meeting about this but it's important to leave a comment here

File: g1ant-language/introduction.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Yes, we could have embraced C, C#, Java, JavaScript, Python, or any other popular language. We could have, but we didn’t — on purpose. We decided to create a whole new experience for those who want just to control robot processes, not to write apps, games or websites. Our aim was to develop a language that would be easy to understand, easy to write, easy to debug. A perfect language that would offer all you need to efficiently create robot tasks and nothing you don’t. It would also allow you to add your own commands, because it’s you who knows best what a robot should do for you.
+
+And we are confident G1ANT.Language is the solution we dreamed of.
+
+## What Are Its Advantages?
+
+First of all, it’s tailored specifically for controlling robot. There are no superfluous commands and arguments, only tools that help you robotize repetitive tasks on a computer.
+
+Secondly, G1ANT.Language is an open platform. You can lurk into the code in our SDK and develop your own addons with new language elements. This makes G1ANT.Language a living, flexible, adapting organism, which grows in resources and functions thanks to community and its innovations.
```
Comment (WiktoriaPrusik) [inline g1ant-language/introduction.md]: so G1ANT.Language is a language or a platform?

Comment (Vegebutcher) [reply g1ant-language/introduction.md]: A language platform :smile:

File: g1ant-language/introduction.md (hunk 1 above)
Comment (lfronczyk) [reply g1ant-language/introduction.md]: IMHO it's like MS would create language called "MS". If you would say "MS" no one would know if you mean the company or the language. G1ANT Language is fine for me.

File: g1ant-language/introduction.md (hunk 2 above)
Comment (ChrisPrusik) [reply g1ant-language/introduction.md]: language is language, platform is platform (like robot, addons, marketplace etc)

Comment (Vegebutcher) [reply g1ant-language/introduction.md]:
```suggestion
Secondly, G1ANT.Language is open to your ideas. You can lurk into the code in our SDK and develop your own addons with new language elements. This makes G1ANT.Language a living, flexible, adapting organism, which grows in resources and functions thanks to community and its innovations.
```

File: g1ant-language/introduction.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+It is a programming language created especially for our robot. If you have some programming knowledge, you will find many similarities to existing languages, since G1ANT.Language shares the same principles and logic. But if you are not familiar with any code and your programming experience is second to none, worry not: it was designed with humans in mind, not robots. Its purpose is to allow you to start robot process automation (*aka* RPA) as soon as possible. You should master G1ANT.Language within hours, if not minutes — and we are here to help you with that.
+
+## Why New Language?
+
+Yes, we could have embraced C, C#, Java, JavaScript, Python, or any other popular language. We could have, but we didn’t — on purpose. We decided to create a whole new experience for those who want just to control robot processes, not to write apps, games or websites. Our aim was to develop a language that would be easy to understand, easy to write, easy to debug. A perfect language that would offer all you need to efficiently create robot tasks and nothing you don’t. It would also allow you to add your own commands, because it’s you who knows best what a robot should do for you.
```
Comment (lfronczyk) [inline g1ant-language/introduction.md]: "and nothing you don’t" - I don't understand this part

**Comment by lfronczyk** [comment]
No changes for last two weeks, I think it might be merged (after considering my remark) and closed.

File: g1ant-language/introduction.md (hunk 3 above)
Comment (Vegebutcher) [reply g1ant-language/introduction.md]: all you need...and nothing you don't (need)
It's a common verbal shortcut

**Pull request merged**